### ISSUE TITLE
Output logs to the terminal

### ISSUE THREAD

**Issue by ymichaelson** [microsoft/vscode-makefile-tools#220]
How can I set output logs to the terminal?
![image](https://user-images.githubusercontent.com/41182093/139567713-0038aa27-e499-4b3e-a0f9-1e73614358c5.png)

**Comment by andreeis** [comment]
@ymichaelson, currently there is nothing designed in VSCode itself to achieve this. The output channel contains anything the extension logs explicitly and the terminal shows the output of any commands that the extension launches there.

What is the scenario where you need this? Is this for build? Luckily, in order to support problem matchers for our next release, we are working now on having the build be run in the terminal.

Thinking further, I imagine we can have a setting where anything we spawn we can launch in the terminal and skip catching and logging any resulting output into the channel window (leave it be shown in the terminal only).

We may not be able to implement this part immediately and we will follow the reactions from the community to evaluate where this last part (not related to build which will be run in terminal starting with the next release) better fits in our backlog.

**Comment by ymichaelson** [comment]
> @ymichaelson, currently there is nothing designed in VSCode itself to achieve this. The output channel contains anything the extension logs explicitly and the terminal shows the output of any commands that the extension launches there.
> 
> What is the scenario where you need this? Is this for build? Luckily, in order to support problem matchers for our next release, we are working now on having the build be run in the terminal.
> 
> Thinking further, I imagine we can have a setting where anything we spawn we can launch in the terminal and skip catching and logging any resulting output into the channel window (leave it be shown in the terminal only).
> 
> We may not be able to implement this part immediately and we will follow the reactions from the community to evaluate where this last part (not related to build which will be run in terminal starting with the next release) better fits in our backlog.

Yes, it just for build, because output to the terminal will look beautiful, and it will be easier to find the key output exactly.
